content/docs/learning-guide/tutorials/postfx/use-postfx-shape-weight-modifier.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Use PostFX Shape Weight Modifier to Modify Camera Exposure
+linktitle: Use PostFX Shape Weight Modifier
+description: Use PostFX Shape Weight Modifier to modify exposure control in Open 3D Engine (O3DE).
+toc: true
+---
+
+In this example, you will set up a spherical post-processing volume that controls the camera's exposure. When the camera moves in and out of the volume, its exposure changes.
+
+Post-processing volumes require two entities: one that defines the PostFX and one that defines a PostFX volume. This example uses the following entities:
+
+- **PostFX**: This entity contains an **Exposure Control** component and a **PostFX Layer** component that set the default amount of light that the camera receives from the environment.
+  
+- **PostFX Volume**: This entity contains a **PostFX Shape Weight Modifier** component and a **Sphere Shape** component that define a PostFX volume. It also contains an **Exposure Control** component and a **PostFX Layer** component that adjust the camera exposure when the camera moves through the PostFX Volume.
+
+To use post-processing volumes: 
+
+1. In **O3DE Editor**, create a default level.
+
+2. Create an entity for the PostFX. Right click in **Perspective** and select **Create entity**.
+  
+3. In the new PostFX entity, add the following components:
+    
+    - PostFX Exposure Control
+    
+    - PostFX Layer
+
+4. In the Exposure Control component, set the **Manual Compensation** property to `1.0`. This sets a default, bright exposure for the camera.
+
+
+5. Create an entity for the PostFX volume. Right click in **Perspective** near the **Shader Ball** entity, and select **Create entity**.  
```

Comment:
Before these steps, simply describe setting up the default PostFX look.
Create an Entity
Add the PostFX Layer component, set the category type to 'default'
Add an Eye Adaptation component

(that just helps the user understand that there is a default look at the bottom of the layer stack, that they can control and manage separately from the other look bound to the volume.)